package.json

```diff
@@ -93,15 +93,11 @@
     "webpack-cli": "^3.3.12"
   },
   "dependencies": {
-    "custom-event": "^1.0.1",
     "dayjs": "^1.8.34",
     "domready": "^0.3.0",
     "es6-promise": "^4.2.8",
     "global": "^4.3.0",
-    "json3": "^3.3.2",
-    "jsonp": "0.2.1",
-    "lit-element": "^2.3.1",
-    "lit-html": "^1.2.1",
+    "jsonp": "^0.2.1",
```

Comment:
Why not remove jsonp?

Reply:
we still have server side cookie, which has not been implemented to support the Fetch API